### ISSUE TITLE
Bug - DescriptionList - help text cannot be triggered via keyboard

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#11467]
**Describe the problem**
The term help text on description list cannot be triggered via keyboard, only via mouse.

**How do you reproduce the problem?**
Go to the term help text example and try opening the popover via keyboard Enter or Space, notice that it does not work.

**Expected behavior**
The term help text can be triggered via mouse or keyboard.

**Is this issue blocking you?**
List the workaround if there is one.

**Screenshots**
If applicable, add screenshots to help explain the issue.

**What is your environment?**

**What is your product and what release date are you targeting?**

**Any other information?**

**Comment by thatblindgeye** [comment]
Note that an issue was also opened for v5: https://github.com/patternfly/patternfly-react/issues/10350 

Part of this should include making a similar update in v5.

**Comment by thatblindgeye** [comment]
Closing this as a consumer had opened this issue in https://github.com/patternfly/patternfly-react/issues/10846
